mlrun/projects/pipelines.py

```diff
@@ -696,6 +696,51 @@ class _RemoteRunner(_PipelineRunner):
 
     engine = "remote"
 
+    @staticmethod
+    def _prepare_load_and_run_function(
+        source,
+        project_name,
+        save,
+        workflow_name,
+        workflow_spec,
+        artifact_path,
+        workflow_handler,
+        namespace,
+    ):
+        # Creating the load project and workflow running function:
+        load_and_run_fn = mlrun.new_function(
+            name=f"workflow-runner-{workflow_name}",
+            project=project_name,
+            kind="job",
+            image=mlrun.mlconf.default_base_image,
+        )
+        runspec = mlrun.RunObject.from_dict(
```

Comment:
why calling it `runspec` if its a `RunObject`? Shouldn't it be `run_object` or `runobj`?

Reply:
The reason for this comes from  this [line](https://github.com/mlrun/mlrun/blob/development/mlrun/runtimes/base.py#L290)